docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -877,6 +877,9 @@ Note that not listing a host port means that Podman automatically selects one, a
 may be different for each invocation of service. This makes that a less useful option. The
 allocated port can be found with the `podman port` command.
 
+When using `host` networking via `Network=host`, the `PublishPort=` option cannot be used.
+Ports that need to be exposed must be specified on the Pod definition using, for example, `containerPort`.
```

Comment:
The first sentence is rightbut the second make no sense. There is no such thing as "exposed" when using host networks. That depends on your host firewall.

Reply:
Hrmm, I re-ran my test scenario and you are right. I was mixing a lot of concerns trying to figure out how to get things to work and I'm not sure what I was running into running a clean test.

I think my confusion came from the natural feeling with containers of needing to expose the ports that are in use. What made this more confusing was I am trying to run a `.kube` based quadlet talking to a non-container based PostreSQL that is available over localhost. Understanding the difference of perspective of localhost with each configuration wasn't as obvious being new to trying this. 

So I'll remove the second sentence and keep the PR with the first sentence as I would still find that helpful in the docs.